ansible/roles/operator-pipeline/templates/openshift/tasks/cert-project-check.yml

```diff
@@ -30,4 +41,15 @@ spec:
           exit 1
         fi
 
+        PROJECT_URL="$(params.pyxis_url)/v1/projects/certification/id/$CERT_PROJECT_ID"
+        PROJECT=$(curl -k -X GET $PROJECT_URL -H  "accept: application/json" --cert $(params.pyxis_cert_path) --key $(params.pyxis_key_path))
+
+        ISV_PID=$(echo $PROJECT | jq -r '.container.isv_pid')
+
+        if [ -z $ISV_PID ]; then
+          echo "Certification project isv_pid not found."
+          exit 1
+        fi
+
         echo -n $CERT_PROJECT_ID | tee $(results.certification_project_id.path)
+        echo -n $ISV_PID | tee $(results.isv_pid.path)
```

Comment:
Why do we need isv_pid?

Reply:
isv_pid is inputted into the reserve-operator-name task as the "assocation"